### ISSUE TITLE
gatsby-plugin-manifest 2.0.28 no longer generates manifest.webmanifest

### ISSUE THREAD

**Issue by pgilad** [gatsbyjs/gatsby#13264]
Looks like something has changed with the plugin and it no longer generates the `public/manifest.webmanifest` file, causing deployments to be messed up 😢 

```bash
% npx gatsby info

  System:
    OS: macOS 10.14.4
    CPU: (4) x64 Intel(R) Core(TM) i7-6567U CPU @ 3.30GHz
    Shell: 3.0.2 - /usr/local/bin/fish
  Binaries:
    Node: 11.13.0 - ~/.nvm/versions/node/v11.13.0/bin/node
    npm: 6.9.0 - ~/.nvm/versions/node/v11.13.0/bin/npm
  Languages:
    Python: 2.7.16 - /usr/local/bin/python
  Browsers:
    Chrome: 73.0.3683.103
    Firefox: 66.0
    Safari: 12.1
  npmPackages:
    gatsby: ^2.3.16 => 2.3.16
    gatsby-cli: ^2.5.6 => 2.5.6
    gatsby-image: ^2.0.37 => 2.0.37
    gatsby-plugin-canonical-urls: ^2.0.12 => 2.0.12
    gatsby-plugin-catch-links: ^2.0.13 => 2.0.13
    gatsby-plugin-feed: ^2.1.0 => 2.1.0
    gatsby-plugin-google-analytics: ^2.0.18 => 2.0.18
    gatsby-plugin-manifest: ^2.0.28 => 2.0.28
    gatsby-plugin-netlify: ^2.0.13 => 2.0.13
    gatsby-plugin-offline: ^2.0.25 => 2.0.25
    gatsby-plugin-react-helmet: ^3.0.12 => 3.0.12
    gatsby-plugin-robots-txt: ^1.4.0 => 1.4.0
    gatsby-plugin-sharp: ^2.0.33 => 2.0.33
    gatsby-plugin-sitemap: ^2.0.12 => 2.0.12
    gatsby-plugin-typescript: ^2.0.12 => 2.0.12
    gatsby-remark-autolink-headers: ^2.0.16 => 2.0.16
    gatsby-remark-copy-linked-files: ^2.0.11 => 2.0.11
    gatsby-remark-embed-gist: ^1.1.7 => 1.1.7
    gatsby-remark-emojis: ^0.3.2 => 0.3.2
    gatsby-remark-images: ^3.0.10 => 3.0.10
    gatsby-remark-prismjs: ^3.2.8 => 3.2.8
    gatsby-source-filesystem: ^2.0.29 => 2.0.29
    gatsby-transformer-remark: ^2.3.8 => 2.3.8
    gatsby-transformer-sharp: ^2.1.17 => 2.1.17
```

**Comment by DSchau** [comment]
Thanks for alerting us of this!

It looks like it was specifically introduced with `2.0.27`, so if you pin to `2.0.26` you can restore the functionality.

Figuring out what the issue is--then will create a fix and publish as soon as possible. Thanks!

**Comment by pgilad** [comment]
Thanks, that's exactly what I did for the time being!

**Comment by DSchau** [comment]
It's this commit: [01670e18d02e08543b8c9e1742b3caa123e91b4e](https://github.com/gatsbyjs/gatsby/commit/01670e18d02e08543b8c9e1742b3caa123e91b4e#diff-0cee23a635d1c08646f35342b6dee83f)

Not really sure why yet!

**Comment by sidharthachatterjee** [comment]
Fixed and published in `gatsby-plugin-manifest@2.0.29`

**Comment by pgilad** [comment]
Thanks! 🏆
